local-notifications/android/src/main/java/com/capacitorjs/plugins/localnotifications/LocalNotificationSchedule.java

```diff
@@ -37,16 +37,19 @@ public LocalNotificationSchedule() {}
     private void buildEveryElement(JSObject schedule) {
         // 'year'|'month'|'two-weeks'|'week'|'day'|'hour'|'minute'|'second';
         this.every = schedule.getString("every");
+        if (this.every != null) {
+            this.repeats = true;
+        }
```

Comment:
What is this change for?

Reply:
In my mind, having a schedule that states something will happen every X implies that this is a repeating notification.

Reply:
According to the docs, repeats is only used for at, so I don’t think we should change its value to something different than the user set. And also the change is unrelated to the original intent of the PR, I would try to avoid unrelated changes. If not setting repeats to true here can cause issues with every, it should be reported and evaluated in a separate issue.